### PR TITLE
Added option to disable motors on cancel

### PR DESCRIPTION
**Pull request by VoltexRB**
Added variable to _CLIENT_VARIABLE to select if motors should be disabled after cancelling a print and added if statement to evaluate variable in CANCEL_PRINT.

Note: I am tired of having to add this on every update, I can't remember a single time I cancel a print and not disable motors. its an optional change.

### CODE REVIEW

**Review by meteyou — APPROVED** [review]

File: client.cfg [hunk 1]
```diff
@@ -20,23 +20,24 @@
...
+#variable_custom_park_y          : 0.0   ; custom y position; value must be within your defined min and max of Y
+#variable_custom_park_dz         : 2.0   ; custom dz value; the value in mm to lift the nozzle when move to park position
+#variable_retract                : 1.0   ; the value to retract while PAUSE
+#variable_cancel_retract         : 5.0   ; the value to retract while CANCEL_PRINT
+#variable_speed_retract          : 35.0  ; retract speed in mm/s
+#variable_unretract              : 1.0   ; the value to unretract while RESUME
+#variable_speed_unretract        : 35.0  ; unretract speed in mm/s
+#variable_speed_hop              : 15.0  ; z move speed in mm/s
+#variable_speed_move             : 100.0 ; move speed in mm/s
+#variable_park_at_cancel         : False ; allow to move the toolhead to park while execute CANCEL_PRINT [True/False]
```
Comment (zellneralex) [inline client.cfg:34]: I do not like that you splitt the pork cancel parameters. Please place your parameter before line 38

**Review by zellneralex — COMMENTED** [review]
See my note about the parameter location. The Readme.md would also need to be enhanced.
Change the examples to include your Parameter. 
Add a section to explain what it does at the bottom.

**Review by zellneralex — CHANGES_REQUESTED** [review]
See my note about the parameter location. The Readme.md would also need to be enhanced.
Change the examples to include your Parameter.
Add a section to explain what it does at the bottom.

**Comment by alcor6502** [comment]
Thanks to [VoltexRB](https://github.com/VoltexRB), I come up with a broader solution which can accomodate other needs not only to just disable the motors.
[Added option to run a custom macro on CANCEL_PRINT #25](https://github.com/mainsail-crew/mainsail-config/pull/25)

**Comment by VoltexRB** [comment]
Better to use pull request by alcor6502

**Pull request closed without merging**